tutorial/README.md

```diff
@@ -0,0 +1,281 @@
+                    <ts:string xml:lang="en">Amount to create</ts:string>
+                </ts:name>
+                <ts:origins>
+                    <!-- e18 is a hard coded multiplier.
+                    rationale for hardcoding: avoiding over-design -->
+                    <ts:user-entry as="e18"/>
+                </ts:origins>
+            </ts:attribute-type>
+            ...
+```
+
+  * Coin.xml: Lastly add the transaction (refers to ethereum smart contract, function, and params in the data tag)
+```
+            <ts:transaction>
+                <ts:ethereum function="mint" contract="Coin">
+                    <ts:data>
+                        <ts:address ref="create-address"/>
+                        <ts:uint256 ref="create-amount"/>
+                    </ts:data>
+                </ts:ethereum>
+            </ts:transaction>
+```
+* Download the xml, css, shtml files, and jump to the section below to test: `Combining files and testing`
+  * The required `Makefile` for combining these files is in `tutorial-files`
+* Next steps: Once you have been able to test the Mint action, you'll notice the transaction is sent, but does not succeed because the wallet is not the minter.
+  * Add a function in the smart contract to set the minter address with a parameter, call from remix with the wallet address
+  * Alternately, add a standard `approve` function in the smart contract, and duplicate/modify the `mint` action in `Coin.xml`.
+
+### [future] Obtaining sample files (schema 2020/03)
+
+A really good starting point to generating your own TokenScript (for the upcoming 2020/03 schema) is available with this [ABI-to-TokenScript tool](https://alphawallet.github.io/ABI-to-TokenScript/):
+
+* Enter the contract address
+* From remix, copy the contract abi, and paste it in the ABI-to-TokenScript tool
+* Enter a contract name, eg "Coin"
+* Finally select erc20 (although this example from studio doesn't completely conform to the erc20 interface)
+* Click `Create your TokenScript!`, and download `Coin.zip`
+
+Inside you will find:
+* A shared.css file
+* javascript files of the form <name>.<lang>.js (about.en.js, approve.en.js)
+* The precursor to your tokenscript file: Coin-TokenScript.xml
+  * update `<ts:address network="1">...contract_address...` to ropsten `<ts:address network="3">...contract_address...`
+  * Rename this file to Coin.xml for simplicity
+* A Makefile to combine the above files
+
+In the following section we'll combine these files in the xml.
+
+### Combining files and testing
+
+Disclaimer: iOS users can simply airdrop the set of js/css/xml files to their device.
+
+To combine the files you will need at least xmllint(ref) to combine the files, this will give you an unsigned (.canonicalized.xml) file for testing.
+When you want to have the file associated with your domain's SSL key, you will then need xmlsectool(ref) create a signed file (.tsml).
+
+* download/install the xmllint command line tool (ref)
```

Comment:
link to more comprehensive instructions?

Reply:
Completed `ref` placeholder